src/content/en/updates/2018/10/background-fetch.md

```diff
@@ -0,0 +1,448 @@
+      <td><code>string</code><br>uniquely identifies this background fetch.
+      <p><code>backgroundFetch.fetch</code> will reject the ID matches an existing background
+      fetch.</p></td>
+    </tr>
+    <tr>
+      <td><code>requests</code></td>
+      <td><code>Array&lt;<a
+      href="https://developer.mozilla.org/en-US/docs/Web/API/Request">Request</a>|string&gt;</code>
+      <br>The things to fetch. Strings will be treated as URLs, and turned into
+      <code>Request</code>s via <code>new Request(theString)</code>.
+      <p>You can fetch things from other origins as long as the resources allow it via
+      <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS">CORS</a>.</p>
+      </td>
+    </tr>
+    <tr>
+      <td><code>options</code></td>
+      <td>An object which may include the following:</td>
+    </tr>
+    <tr>
+      <td><code>options.title</code></td>
+      <td><code>string</code><br>A title for the browser to display along with progress.</td>
+    </tr>
+    <tr>
+      <td><code>options.icons</code></td>
+      <td><code>Array&lt;<a
+      href="https://developer.mozilla.org/en-US/docs/Web/Manifest">IconDefinition</a>&gt;</code><br>
+      An array of objects with a `src`, `size`, and `type`.</td>
+    </tr>
+    <tr>
+      <td><code>options.downloadTotal</code></td>
+      <td><code>number</code><br>The total size of the response bodies (after being un-gzipped).
+      <p>Although this is optional, it's strongly recommended that you provide it. It's used to tell
+      the user how big the download is, and to provide progress information. If you don't provide
+      this, the browser will tell the user the size is unknown, and as a result the user may be more
+      likely to abort the download.</p>
+      <p>If the background fetch downloads exceeds the number given here, it will be aborted. It's
+      totally fine if the download is smaller than the <code>downloadTotal</code>, so if you aren't
+      sure what the download total will be, it's best to err on the side of caution.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+`backgroundFetch.fetch` returns a promise that resolves with a `BackgroundFetchRegistration`. I'll
+cover the details of that later. The promise rejects if the user has opted out of downloads, or one
+of the provided parameters is invalid.
+
+Providing many requests for a single background fetch lets you combine things that are logically a
+single thing to the user. For example, a movie may be split into 1000s of resources (typical with
+[MPEG-DASH](https://developer.mozilla.org/en-US/docs/Web/Apps/Fundamentals/Audio_and_video_delivery/Live_streaming_web_audio_and_video#MPEG-DASH)),
+and come with additional resources like images. A level of a game could be spread across many
+JavaScript, image, and audio resources. But to the user, it's just "the movie", or "the level".
+
+Caution: Chrome's implementation currently only accepts requests without a body. In future, bodies
```

Comment:
Is there also a restriction in place around `GET` vs. other verbs? Like, what would happen right now if a user sends a body-less `DELETE` request?

Reply:
That's a good question. Pretty sure it's `GET` only, but I'll check.